src/transformState.js

```diff
@@ -5,7 +5,27 @@
  * @param {Object[]} actions
  */
 function transformState(state, actions) {
-  // write code here
+  for (const element of actions) {
+    for (const key in element) {
```

Comment:
It isn't a good idea to use loop `for` inside another loop `for`
You can remove the second loop